app/interactors/delete_refresh_token.rb

```diff
@@ -0,0 +1,21 @@
+class DeleteRefreshToken
+  include Interactor
+
+  delegate :token, to: :context
+
+  def call
+    context.fail!(error_data: error_data) unless refresh_token
+
+    refresh_token.delete
```

Comment:
Should we use destroy method?
```suggestion
    refresh_token.destroy
```